deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/JDKImperativeDeckOfCardsAsListTest.java

```diff
@@ -142,12 +150,25 @@ public void cardsBySuitIsImmutable()
     @Test
     public void countsBySuit()
     {
-        Assertions.assertEquals(Long.valueOf(13), this.jdkDeck.countsBySuit().get(Suit.CLUBS));
+        Map<Suit, Long> map = new HashMap<>();
+        map.put(Suit.CLUBS, 13L);
+        map.put(Suit.DIAMONDS, 13L);
+        map.put(Suit.HEARTS, 13L);
+        map.put(Suit.SPADES, 13L);
+        Assertions.assertEquals(map, this.jdkDeck.countsBySuit());
     }
 
     @Test
     public void countsByRank()
     {
-        Assertions.assertEquals(Long.valueOf(4), this.jdkDeck.countsByRank().get(Rank.TEN));
+        Map<Rank, Long> map = new HashMap<>();
```

Comment:
You can iterate over all of the `values()` of `Rank` and put them in a `Map` with a value of `4L` each and then compare that to `this.jdkDeck.countsByRank()`.